woocommerce-zaver-checkout.php

```diff
@@ -228,9 +228,17 @@ private function __construct() {
 
 		add_action( 'init', array( $this, 'load_textdomain' ) );
 
-		$this->logger        = new Logger( 'zaver_checkout', 'Zaver Checkout' );
-		$this->system_report = new SystemReport( 'zaver_checkout', 'Zaver Checkout', array() );
+		$included_settings   = array(
+			array(
+				'type'       => 'title',
+				'is_section' => true,
+			),
+			array( 'type' => 'checkbox' ),
+		);
+		$this->system_report = new SystemReport( 'zaver_checkout', 'Zaver Checkout', $included_settings );
+		$this->logger        = new Logger( 'zaver_checkout', wc_string_to_bool( $settings['logging'] ?? false ) );
 		$this->session       = new Classes\Session();
+
```

Comment:
I think this is overriding things that another branch had merged into the current main branch. Maybe we should update the logging branch from main, and make this PR go to that branch instead?